AnimationSystem/AnimationSystem.cc

```diff
@@ -593,7 +596,7 @@ namespace AnimationSystem {
       this->randomIdleState = true;
       this->randomIdleDuration = j["duration"];
       this->randomIdleSpeed = j["speed"];
-      this->randomIdleTransition = j["transition"];
+      this->randomIdleStartTime = j["startTime"];
```

Comment:
They are `randomIdleStartTimeS` and `startTimeS`.
Best to keep the convention that ending `S` means `seconds`, without ending `S` means `milliseconds`.